applications/nrf5340_audio/src/modules/led.c

```diff
@@ -184,6 +184,11 @@ static int led_device_tree_parse(void)
 				return ret;
 			}
 
+		} else if (strstr(led_labels[i], "LED_MONO_BLUE") && CONFIG_AUDIO_DEV == HEADSET) {
+			/* Not initialize LED_MONO_BLUE here on headset devices.
```

Comment:
Suggest removing this section, leaving LED_MONO_BLUE to be initialized and set for all other cases than the blinking

Reply:
I had some problems when "LED_MONO_BLUE" was initialized as an GPIO and wanted to use the same pin as an GPIOTE. 
That is also the reason why the functionality for this led is different on headsets and gateway. 
I will take closer look at this now and will give an update on this post.

Reply:
After looking into this the documentation, I found out that when a pin is configured as an GPIOTE it cannot be controlled as an normal GPIO.